pkgs/applications/version-management/git-and-tools/ghq/default.nix

```diff
@@ -1,29 +1,31 @@
-{ stdenv, buildGoPackage, fetchFromGitHub }:
+{ stdenv, buildGoModule, fetchFromGitHub }:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "ghq";
   version = "0.12.6";
 
-  goPackagePath = "github.com/motemen/ghq";
-
   src = fetchFromGitHub {
     owner = "motemen";
     repo = "ghq";
     rev = "v${version}";
     sha256 = "14rm7fvphr7r9x0ys10vhzjwhfhhscgr574n1i1z4lzw551lrnp4";
   };
 
-  goDeps = ./deps.nix;
+  modSha256 = "1jbhd25nxl6yckwp9z2nfz2qzv76kg70b43286r26v4ypgpx3ys4";
 
   buildFlagsArray = ''
     -ldflags=
       -X=main.Version=${version}
   '';
 
   postInstall = ''
-    install -m 444 -D ${src}/zsh/_ghq $bin/share/zsh/site-functions/_ghq
+    install -m 444 -D ${src}/zsh/_ghq $out/share/zsh/site-functions/_ghq
```

Comment:
Shall we add a `bin` output here for backwards compatibility?
Putting this in `bin` seems very wrong to me

Reply:
Due to historical reasons we have `$bin` in `buildGoPackage` as `$out` contained the source code of it. At some point we got rid of the source code but kept `$bin` for compatibility reasons. We corrected this when introducing `buildGoModule`.